lib/simple-seeder.js

```diff
@@ -238,6 +244,8 @@ module.exports = class SimpleSeeder {
 
     if (opts.seeders) info.seeders = await this._createSeeders(publicKey)
 
+    this.allowedPeers = await list.metadata.get(allowedPeersKey)
```

Comment:
The code will work but I would like this internal lib to not assume one single list, but here we're overwriting the `allowedPeers` if we add multiple lists

Simple solution is to add a `userData` below over here:
https://github.com/holepunchto/simple-seeder/blob/daefead02f5157e723e25362560421d69862abcb/lib/simple-seeder.js#L49

And then instead of `this.allowedPeers = ..` is just:
`info.userData = ..`
Or
`info.userData = { allowedPeers: .. }`